rag_apps/youtube_transcript_rag/app.py

```
@@ -0,0 +1,362 @@
+    chroma = get_chroma_client()
+    emb_fn = get_embedding_fn()
+    cname = collection_name_for(video_id)
+
+    with tempfile.TemporaryDirectory() as tmp:
+        status_widget.write("⬇️ Downloading audio…")
+        audio_path = download_audio(url, tmp)
+
+        status_widget.write("🎙️ Transcribing with Whisper (this can take a few minutes for long videos)…")
+        segments = transcribe_audio(audio_path)
+
+    status_widget.write(f"✂️ Splitting into chunks…")
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)